### ISSUE TITLE
loss 在第二个step就变成了0

### ISSUE THREAD

**Issue by TAOSHss** [QwenLM/Qwen-VL#352]
loss 在第二个step 就变成了0 训练依旧可以继续运行；这个情况 导致我 不能够了解 训练的情况，出现这种情况的原因可能是什么呢，数据的准备我剔除了较大的图像以及较小的图像，训练的图像大小都在448*448附近；并且都是单轮对话
`[
  {
    "id": "identity_0",
    "conversations": [
      {
        "from": "user",
        "value": "Picture 1: <img>5019b9e77382910074.png</img>\n<ref>text1</ref>"
      },
      {
        "from": "assistant",
        "value": "<box>(222,333),(444,555)</box>"
      },
      
    ]
  },`
            

`
{'loss': 3.8462, 'learning_rate': 3.4482758620689656e-07, 'epoch': 0.0}
{'loss': 0.0, 'learning_rate': 6.896551724137931e-07, 'epoch': 0.01}
{'loss': 0.0, 'learning_rate': 1.0344827586206898e-06, 'epoch': 0.01}
{'loss': 0.0, 'learning_rate': 1.3793103448275862e-06, 'epoch': 0.01}
{'loss': 0.0, 'learning_rate': 1.724137931034483e-06, 'epoch': 0.02}
`

**Comment by jackyjinjing** [comment]
用v100训练的也遇到了同样的问题，换a100就可以了，应该是数据类型的问题，v100当时只能用float16，然后a100换成bf16就好了

**Comment by TAOSHss** [comment]
我训练的时候，只能用fp32，用不了fp16和bf16。感谢你的解答



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年4月7日(周日) 晚上6:49
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [QwenLM/Qwen-VL] loss 在第二个step就变成了0 (Issue #352)




 
用v100训练的也遇到了同样的问题，换a100就可以了，应该是数据类型的问题，v100当时只能用float16，然后a100换成bf16就好了

**Comment by typhoon1104** [comment]
我也遇到了这个问题，楼主解决了吗？

**Comment by aixiaoing** [comment]
这个确实是卡的问题，V100用fp16或者fp32都不行，都会loss为0，换了A100就OK了，猜测就是需要卡能支持bf16

**Comment by boyugou** [comment]
原来如此，H100不知道为什么也不行
